### DISCUSSION TITLE
请问 G6 5.0 的正式版发布计划是什么？

### DISCUSSION THREAD

**Discussion by AlichChen** [antvis/G6#5287] [Q&A] [answered]
G6 5.0 在功能和性能上都有较大提升，但当前beta版本的稳定性还无法在生产系统中使用，但项目前期需要确定 G6 版本，希望了解下一个大致的发布计划，如果已临近发布节点，则考虑直接使用 5.0,

**Comment by Aarebecca** [comment, marked as answer]
预计近期会有一定 API 和 Spec 的调整，在 2月底左右会完成相关改造。正式版计划于 2024年6月份发布。

**Reply by AlichChen** [reply]
谢了

**Comment by hanfengcan** [comment]
现在5月份了，V5的版本稳定下来了吗

**Reply by hustcc** [reply]
本周会部署官网，验证视觉和交互，然后跟进 bugfix，到 6 月发布稳定版本。
